NetworkOfPrivateClinics/NetworkOfPrivateClinics/GenerateData.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NetworkOfPrivateClinics
+{
+    public static class GenerateData
```

Comment:
Let's name it `ClinicRepository` because it looks like implementing repository pattern (in memory storage). Read a bit about repository pattern.
Remove all static fields and make it normal class. Static fields are not needed here.
We use static fields for constants, small utilities, readonly shared data, extension methods, etc. In other cases they are bad because:
1. it's not thread safe
2. it's difficult to unit test